core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -229,8 +229,9 @@ public Jdbi installPlugins() {
      * @return this
      */
     public Jdbi installPlugin(JdbiPlugin plugin) {
-        Unchecked.consumer(plugin::customizeJdbi).accept(this);
-        plugins.add(plugin);
+        if (plugins.addIfAbsent(plugin)) {
```

Comment:
I can see this causing issues for some people since behavior in jdbi can depend a lot on registration order. If someone does duplicate plugin registrations, this change will break things for them. Is it acceptable in this case?

Reply:
I think that's okay.  Repeatedly installing a plugin just to change the ordering is pretty broken to start with.  Also, plugins do not currently implement `equals` -- so it would have to be the same *instance* to trigger this check.  I think the risk is small.